### ISSUE TITLE
Add support for keep-alive-time parameter

### ISSUE THREAD

**Issue by cescoffier** [eclipse-vertx/vertx-grpc#74]
The Netty variant of the gRPC server allows configuring the keep-alive-time. However, the Vert.x variant does not. 

Related to: https://github.com/quarkusio/quarkus/issues/36033

**Comment by vietj** [comment]
which keep alive are we talking about ? HTTP/2 ? it is not clear

**Comment by cescoffier** [comment]
In the NettyChannelBuilder, you have the `io.grpc.netty.NettyChannelBuilder#keepAliveTime` with the following javadoc:

> Sets the time without read activity before sending a keepalive ping. An unreasonably small value might be increased, and Long.MAX_VALUE nano seconds or an unreasonably large value will disable keepalive. Defaults to infinite.
Clients must receive permission from the service owner before enabling this option. Keepalives can increase the load on services and are commonly "invisible" making it hard to notice when they are causing excessive load. Clients are strongly encouraged to use only as small of a value as necessary

Basically the time between _pings_. 

Do we have the equivalent method in the pure Vert.x variant?

**Comment by vietj** [comment]
assuming HTTP/2 ping ?

**Comment by cescoffier** [comment]
Yes, Http/2 pings.

**Comment by vietj** [comment]
It seems that it periodically sends a ping HTTP/2 frame to the server to "keep alive" the stream. We could develop a similar feature for the new client on top of Vert.x HTTP/2 connection ping feature.

**Comment by ThoSap** [comment]
Is this still planned to be worked on?

**Comment by vietj** [comment]
it is actually not defined what this means, if that is defined then we can tell whether or not it will be planned

**Comment by sigurd-cp** [comment]
> it is actually not defined what this means, if that is defined then we can tell whether or not it will be planned

I would expect, that a `keep-alive-timeout` can be configured and based on this configuration a ping is sent to keep the connection alive (e.g. a grpc server, also a load balancer, getting informed and does not cancel the connection).

FYI: There is also an open issue in the quarkus project https://github.com/quarkusio/quarkus/issues/36033 

Please let us know what is still undefined.
